### PR TITLE
fpga: dfl: Add vendor IDs and device IDs for ALIBABA F5 cards

### PR DESCRIPTION
**Pull request by richardli001**
This patch adds the approved PCI Express Vendor IDs and Device IDs for the ALIBABA cards.

Signed-off-by: richard.li <richard.li@linux.alibaba.com>

### CODE REVIEW

File: include/linux/pci_ids.h [hunk 1]
```diff
@@ -1366,6 +1366,8 @@
 #define PCI_DEVICE_ID_IMS_TT128		0x9128
 #define PCI_DEVICE_ID_IMS_TT3D		0x9135
 
+#define PCI_VENDOR_ID_ALIBABA		0x1ded
```
Comment (rweight) [inline include/linux/pci_ids.h:1369]: I see that this definition is also present in drivers/infiniband/hw/erdma/erdma_hw.h. Have you considered removing the duplicate definition from that location?

**Review by rweight — COMMENTED** [review]
Thanks for the patch! This looks like a patch that could be submitted upstream as there are no dependencies on code that is not upstream. If you submit this upstream now, we would pick up the changes when we rebase to 6.2-rc1, likely at the end of December and it would be propagated to the appropriate branches, including the linux-dfl-backport project.

If you need this patch to be integrated into linux-dfl sooner than that, please reply to let me know and I will pull it into the linux-dfl branches. We will test it internally first, and then it will be pushed to the fpga-ofs-dev-5.15-lts and fpga-ofs-dev branches.

File: include/linux/pci_ids.h (hunk 1 above)
Comment (richardli001) [reply include/linux/pci_ids.h:1369]: Duplication do not influence erdma's compile and work now, but I think it will be removed in the next patch.

**Comment by richardli001** [comment]
It will be appreciate if the patch could be pushed to fpga-ofs-dev-5.15-lts and fpga-ofs-dev branches first.

**Comment by rweight** [comment]
I have taken the patch. It is now present on fpga-ofs-dev-5.15-lts and fpga-ofs-dev.

**Pull request closed without merging**